homeassistant/components/sensor/netatmo.py

```diff
@@ -0,0 +1,115 @@
+"""
+homeassistant.components.sensor.netatmo
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This sensor connects to the cloud service provided by netatmo and uses
+the api.
+To successfully connect it needs the client_id and client_secret from
+a netatmo developer account and your username and password from the
+netatmo user account
+"""
+from datetime import timedelta
+import logging
+import requests
+
+from homeassistant.const import (STATE_UNKNOWN)
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import TEMP_CELCIUS
+
+SENSOR_TYPES = {
+    'Temperature': [TEMP_CELCIUS],
+    'Humidity': ['%'],
+    'CO2': ['ppm'],
+    'Noise': ['dB'],
+    'Pressure': ['mb']
```

Comment:
Hi,
I took that from the Netatmo Web GUI where it says mb, but you are correct, it should be mbar.
I will change that